view/theme/frost-mobile/style.css

```diff
@@ -3990,6 +3982,16 @@ aside input[type='text'] {
 	opacity: 0.5;
 }
 
+.event-icon {
+	display: block; width: 33px; height: 33px;
+	background-size: 100% 100%;
+	background-repeat: no-repeat;
+	opacity: 0.5;
+}
+.event-attend-icon { background-image: url('images/event-attend.png'); }

```

Comment:
Ouf.. I forgot to add them to git.. I don't have them here now.. you have to wait until Wednesday :-)